src/pydata_sphinx_theme/assets/styles/components/_search.scss

```
@@ -14,7 +14,8 @@
   }
 
   i.fa-solid.fa-magnifying-glass {
-    padding-right: 0.5rem;
+    position: absolute;
+    left: 1.5rem;
```

```suggestion
    position: absolute;
    left: 1.6rem;
    color: var(--pst-color-text-muted);
```

slight adjustments for spacing and color